cudamapper/src/index_gpu.cuh

```diff
@@ -715,41 +732,43 @@ void IndexGPU<SketchElementImpl>::generate_index(const io::FastaParser& parser,
 
     // copy the data to member functions (depending on the interface desing these copies might not be needed)
     representations_d_.resize(representations_d.size());
-    if (representations_d_.capacity() > representations_d_.size())
-        representations_d_.shrink_to_fit();
+    //if (representations_d_.capacity() > representations_d_.size())
```

Comment:
Please do not leave code commented. Commented code will never be cleaned up, because nobody is sure if it might still be required for something. Either remove it completely, leave it or add more comment providing context.

Reply:
It seems this should be removed.